src/service-card.js

```diff
@@ -0,0 +1,71 @@
+import { LitElement, html, css } from 'lit-element/lit-element.js';
+
+class ServiceCard extends LitElement {
+  static get tag() {
+    return 'service-card';
+  }
+
+  static get properties() {
+    return {
+      image: { type: String },
+      title: { type: String },
+      info: { type: String },
+      alt: { type: String },
+      uniformPadding: { type: String, attribute: 'uniform-padding' },
+    };
+  }
+
+  constructor() {
+    super();
+    this.title = 'A shorter title';
+    this.info = 'Here is the information about the thing';
+    this.uniformPadding = '2';
+  }
+
+  static get styles() {
+    return css`
+      :host {
+        display: inline-flex;
+      }
+      .card {
+        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+        transition: 0.3s all ease-in-out;
+        width: var(--service-card-card-width, 300px);
```

Comment:
css variable; this is a good way to allow people to modify this if they needed to via CSS